app/assets/stylesheets/sessions.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Sessions controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: https://sass-lang.com/
```

Comment:
![image](https://user-images.githubusercontent.com/65896888/91111549-feb00a00-e6aa-11ea-82b5-d60cb50600f9.png)

Reply:
remove this file because you have not used it